content/workload-reduction-toolkit/evaluate-workload-measures/structured-conversations-with-staff.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Have stuctured conversations with staff to evaluate outcomes
+layout: /markdown_resource.html.erb
+colour: green
+---
+
+# Have structured conversations to evaluate outcomes
+
+<strong class="govuk-tag">Template</strong>
+
+A structured conversation template to help gather feedback from staff and teachers about your workload reduction measures.
+
+## Adapt the template
+
+Edit the template to better reflect your school.
+
+Some school leaders customise this structured conversation template based on the results of their staff workload survey,
+or prior structured conversations with staff.
+
+## Decide when you’ll have the conversations
+
+Consider when in the school year it would be best to have structured conversations.
+
+Some school leaders find it helpful to have multiple structured conversations throughout the year, to continue evaluating
+the impact of their workload reduction measures.
+
+## Decide who will be involved in the conversations
+
+Consider who should:
+
+- lead the conversations, which could be delegated to year group or subject leads
+- participate in the conversations, which could be one-on-one or held in a group involving staff members from different roles
+
+You could also consider repeating the conversations with the same lead and participants from prior structured discussions.
+
+## Plan your next steps
+
+Consider how you will act on the results of the conversations. Give yourself time to analyse the results, update your staff
+and act.
+
+<div class="govuk-grid-row dfe-width-container govuk-!-padding-bottom-6">
```

Comment:
I think this needs to go above "Adapt the template", similar to the other download resources.